Actions summaries

severinbratus opened the pull request:
Resolves #1

rlanzafame commented:
Looks good to me, I am excited to use it! (but don't have time at the moment - I'll let Tom sign off on this one :))

Tom-van-Woudenberg reviewed (changes requested):
Awesome!

I've changed one thing (double / somewhere) and tried implementing something else (but failed): can you give the full link in the aliases table? So https://teachbooks.github.io/.../alias --> https://teachbooks.github.io/.../target instead of only alias --> target

And maybe show the branches deployed table only when the process is finished? Otherwise it might give the impression that the urls are already available, even if it finishes at a later stage.

severinbratus commented:
> Awesome!
> 
> I've changed one thing (double / somewhere) and tried implementing something else (but failed): can you give the full link in the aliases table? So https://teachbooks.github.io/.../alias --> https://teachbooks.github.io/.../target instead of only alias --> target
> 
> And maybe show the branches deployed table only when the process is finished? Otherwise it might give the impression that the urls are already available, even if it finishes at a later stage.

Fixed both in new commit, please see is this summary looks OK: https://github.com/TeachBooks/testable-template/actions/runs/8723981979
